learn/avalanche-bridge-faq.md

```diff
@@ -78,11 +78,11 @@ Note that **you should not directly transfer tokens to these addresses**. You sh
 
 #### How do fees work on the Avalanche Bridge?
 
-The bridge charges transfer fees in order to cover the cost of the transaction fees on the Avalanche and Ethereum networks, as well as the operational costs of the bridge infrastructure. These fees are charged in-kind with the ERC20 asset being transferred. That is, when you transfer a token, a portion of the balance transferred is taken by the AB as a fee.
+The bridge charges transfer fees in order to cover the cost of the transaction fees on the Avalanche and Ethereum networks, as well as the operational costs of the bridge infrastructure. These fees are charged in-kind with the ERC20 asset being transferred. That is, when you transfer a token, a portion of the balance transferred goes towards covering the fee.
 
 When moving assets from Ethereum to Avalanche, the fee is $3 worth of the ERC20 asset being transferred. Transfers to Avalanche may qualify for an AVAX airdrop as described [here](avalanche-bridge-faq.md#airdrop).
 
-When moving assets from Avalanche to Ethereum, the fee is the value of the maximum Ethereum transaction fee (gas limit * current gas price), plus a constant dollar amount \(currently $5\) to account for price volatility. Note that the maximum Ethereum transaction fee is based on the gas limit and may be higher than the actual transaction fee, which is based on the amount of gas used by the transaction.
+When moving assets from Avalanche to Ethereum, the fee is largely based on the **expected** Ethereum transaction fee. This amount is calculated using current asset prices, the current Ethereum gas price, and the approximate amount of gas that will be used by the transaction. As such, this fee amount can be highly variable. In order to account for price volatility, the calculation is padded by a constant dollar amount \(currently $15\). Note that the fee may end up being different than the transaction fee value displayed in explorers such as Etherscan due to asset and gas price fluctuations, as well as the amount of gas actually used by the transaction being different than the approximation. The current expected fee amounts are displayed in the bridge UI when performing a transfer.
```

Comment:
```suggestion
When moving assets from Avalanche to Ethereum, the fee charged by the Avalanche Bridge is largely based on the **expected** Ethereum transaction fee, which is calculated using current asset prices, the current Ethereum gas price, and the approximate amount of gas that will be used by the Ethereum transaction. As such, the Ethereum transaction fee can be highly variable. To account for price volatility, the Avalanche Bridge charges a constant dollar amount \(currently $15\) more than the estimated Ethereum transaction fee. Note that the fee may end up being different than the transaction fee displayed in explorers such as Etherscan because asset prices, the Ethereum gas price, and the amount of gas used by the Ethererum transaction fluctuate. The expected Avalanche Bridge fee is displayed in the bridge UI when performing a transfer.
```

Reply:
`Note that the fee may end up being different than the transaction fee value displayed in explorers`
Is the first `the fee` the Bridge fee or the Ethereum transaction fee? If the fee on an explorer isn't right, where can users find the right fee?

Reply:
The first "the fee" is the bridge fee. The estimated and actual bridge fees are displayed in the bridge UI, but will differ from the amount shown in Etherscan (which is the Ethereum transaction fee).

Reply:
Would you mind adding that to the snippet above? :pray:

Reply:
Updated now. Reworded slightly to be more clear. :+1: